### PR TITLE
tsweb: memoize the string forms of HTTP response codes.

### PR DESCRIPTION
**Pull request by danderson**
Saves 1-2 allocs per HTTP request after warmup, thanks to
avoiding strconv and fmt.

Signed-off-by: David Anderson <danderson@tailscale.com>

### CODE REVIEW

**Review by bradfitz — COMMENTED** [review]
factor it out into a func?

**Comment by danderson** [comment]
Done, PTAL?

File: tsweb/tsweb.go [hunk 1]
```diff
@@ -302,15 +303,38 @@ func (h retHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if h.opts.StatusCodeCounters != nil {
-		key := fmt.Sprintf("%dxx", msg.Code/100)
-		h.opts.StatusCodeCounters.Add(key, 1)
+		h.opts.StatusCodeCounters.Add(responseCodeString(msg.Code/100), 1)
```
Comment (bradfitz) [inline tsweb/tsweb.go:306]: /100*100, no?

Comment (bradfitz) [reply tsweb/tsweb.go:306]: oh, nevermind, I see :)

**Review by bradfitz — APPROVED** [review]

**Pull request merged**